addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2,6 +2,8 @@ odoo.define('website.editor.snippets.options', function (require) {
 'use strict';
 
 const {ColorpickerWidget} = require('web.Colorpicker');
+
+var { openHeadBodyEditor } = require('@web_editor/js/common/ace-dialog');
```

Comment:
In new code, we never use `var` anymore. Either `const` when you can (most of the time), or `let`.